google/google.js

```diff
@@ -250,7 +250,7 @@ module.exports = function(RED) {
             delete credentials.callback;
             RED.nodes.addCredentials(node_id, credentials);
             request.get({
-                url: 'https://www.googleapis.com/plus/v1/people/me',
+                url: 'https://www.googleapis.com/userinfo/v2/me',
```

Comment:
Does this change break existing users ? The old endpoint still seems to respond
(though I haven't tried authenticating against it)

Reply:
This seems to be the identification part not authentication.
Could try catch to be over sure that does not break anything.